src/serverlist.cpp

```diff
@@ -115,9 +115,7 @@ bool deleteEntry(const ServerListSpec &server)
 	std::string path = ServerList::getFilePath();
 	std::ostringstream ss(std::ios_base::binary);
 	ss << ServerList::serialize(serverlist);
-	if (!fs::safeWriteToFile(path, ss.str()))
-		return false;
-	return true;
+	return !fs::safeWriteToFile(path, ss.str()) ? true : false;
```

Comment:
This is the wrong way around. I suggest to revert this change.

Reply:
Can't this simply be `return fs::safeWriteToFile(path, ss.str())`?

Reply:
Yes, but not point in changing the file just for that, which is why I suggested a reversal.